buildpack/buildpack.go

```diff
@@ -18,14 +18,17 @@ package buildpack
 
 import (
 	"fmt"
-	"path/filepath"
-
 	"github.com/buildpack/libbuildpack/buildpack"
 	"github.com/cloudfoundry/libcfbuildpack/logger"
 	"github.com/mitchellh/mapstructure"
+	"path/filepath"
 )
 
-const cacheRoot = "dependency-cache"
+const (
+	cacheRoot            = "dependency-cache"
+	DependenciesMetadata = "dependencies"
+	DefaultVersions      = "default_versions"
+)
```

Comment:
The above exported constants are used in test_build_factory, though making them global would just reduce duplication.